issue-001/solution.js

```diff
@@ -1,3 +1,36 @@
 function dataTypeCheck(a) {
     // implement your solution here
+    type = typeof(a)
```

Comment:
Exclude this file from this branch. Just delete and do another push to this branch, i.e.
```
git checkout 2-drill-1-strict-check
rm issue-001/solution.js
git add -A
git commit -m 'remove files'
git push origin 2-drill-1-strict-check
```

Do the same for the other branches with stray files.